.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I've heard feedback from projects where I've included a .babelrc that it interferes with dependents that also use babel. Babel is somehow pulling in all the .babelrc's even in node_modules. Are you using a require hook, or can you include the config in a build.